### PR TITLE
[WIP] Spring 2018 updates

### PR DESCRIPTION
**Pull request by eldebrim**

### CODE REVIEW

File: devopstheme/templates/index.html [hunk 1]
```diff
@@ -9,7 +9,7 @@
     </div>
     <div class='text'>
         <h1>Devops BootCamp Fall Kickoff</h1>
-        <h2>November 4th, 2017<br>
+        <h2>February 10th, 2018<br>
```
Comment (doublej472) [inline devopstheme/templates/index.html]: This is no longer the date we are using

**Comment by doublej472** [comment]
[DayCamp](https://trello.com/c/kQXsLiLl/691-daycamp)

**Comment by doublej472** [comment]
@ramereth what should I do about [lessons 16-20](http://devopsbootcamp.osuosl.org/)? Currently they are being left out of this version of the schedule.

**Comment by ramereth** [comment]
@doublej472  I guess keep them as something they can do after bootcamp on their own.

**Comment by doublej472** [comment]
@ramereth alright then, merging!

**Pull request merged**